### PR TITLE
4092 - Review comments unavailable on latest cycle

### LINKED ISSUE #4092: Review comments unavailable on latest cycle
Data table doesn't show review comments button , because Authorizer.canViewReview return false.
Logged in as admin 
https://fra-data.review.fao.org/assessments/fra/latest/FIN/sections/growingStockComposition
Image

Even on Prod it doesn't show 
https://fra-data.fao.org/assessments/fra/2025/FIN/sections/growingStockComposition
Image

### PR DESCRIPTION
**Pull request by sorja**
image

Rules:

Reading review comments:

ADMINISTRATOR: all country statuses
REVIEWER, NATIONAL_CORRESPONDENT, ALTERNATE_NATIONAL_CORRESPONDENT, COLLABORATOR: notStarted,review,editing

Posting review comments:
notStarted and editing

### CODE REVIEW

File: src/client/components/MessageCenter/Topic/Footer/hooks/useShowActions.tsx [hunk 1]
```diff
@@ -17,15 +17,17 @@ export const useShowActions = (topic: MessageTopic) => {
...
   const isOpened = topic.status === MessageTopicStatus.opened
   const isResolved = topic.status === MessageTopicStatus.resolved
   const hasMessages = topic.messages?.length > 0
 
-  const canResolve = isEditing && isReview && isOpened && hasMessages && (isAdmin || isReviewer)
+  const canResolve = (isNotStarted || isEditing) && isReview && isOpened && hasMessages && (isAdmin || isReviewer)
 
-  const canPostMessage = (!isReview || isEditing) && (isOpened || (isResolved && (isAdmin || isReviewer)))
+  const canPostMessage =
+    (!isReview || isEditing || isNotStarted) && (isOpened || (isResolved && (isAdmin || isReviewer)))
```
Comment (sorja) [inline src/client/components/MessageCenter/Topic/Footer/hooks/useShowActions.tsx:30]: Updated posting and resolving:
Include country status : notStarted

File: src/meta/user/authorizer/canViewReview.ts [hunk 2]
```diff
@@ -55,16 +55,20 @@ export const canViewReview = (props: AuthProps): boolean => {
...
-      RoleName.ADMINISTRATOR,
       RoleName.REVIEWER,
       RoleName.NATIONAL_CORRESPONDENT,
       RoleName.ALTERNATE_NATIONAL_CORRESPONDENT,
       RoleName.COLLABORATOR,
     ].map((role) => [role, allowedStatuses])
   )
 
+  // Admin can view in all statuses
+  countryStatus[RoleName.ADMINISTRATOR] = Object.values(AssessmentStatus)
```
Comment (sorja) [inline src/meta/user/authorizer/canViewReview.ts:71]: Admin can always view

**Review by minotogna — APPROVED** [review]

File: src/meta/user/authorizer/canViewReview.ts (hunk 2 above)
Comment (minotogna) [reply src/meta/user/authorizer/canViewReview.ts:71]: Can the reviewer see when it's in review status ?

**Review by minotogna — APPROVED** [review]

File: src/meta/user/authorizer/canViewReview.ts (hunk 2 above)
Comment (sorja) [reply src/meta/user/authorizer/canViewReview.ts:71]: Maybe you found the answer already, but yes!

```
RoleName.REVIEWER: [AssessmentStatus.notStarted, AssessmentStatus.review, AssessmentStatus.editing]
```

**Pull request merged**